Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,12 +1,52 @@
 ﻿using System;
+using System.Globalization;
 
 namespace BootCamp.Chapter
 {
     class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            string name;
+            string askAgain;
+            int age;
+            int weight;
+            int height;
```

Comment:
You want to declare those variables as late as possible. This means at the very moment of first usage.
Also having the type float for weight and height is probably better.